install.sh

```
@@ -398,29 +398,65 @@ install_maina() {
+      exit 10
       ;;
   esac
 }
 
+# Emits the shell-profile hint for the user to add `~/.bun/bin` (or equivalent)
+# to PATH. Picks the user's shell profile heuristically.
+path_hint() {
+  local shell_name="${SHELL##*/}"
+  local profile
+  case "$shell_name" in
+    zsh)  profile="~/.zshrc" ;;
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)